include/lsst/meas/base/FlagHandler.h

```diff
@@ -39,6 +39,15 @@ namespace lsst { namespace meas { namespace base {
     C-strings so we can create these arrays using initializer lists even in C++98.
  */
 struct FlagDefinition {
+
+    FlagDefinition() {
+    }
+
+    FlagDefinition(char const * _name, char const * _doc) {
+        name = _name;
+        doc = _doc;
+    }
+
```

Comment:
This would indeed cause problems if called from Python.  But the reason we needed to use raw pointers instead of `std::string` (direct initialization of an array of struts containing strings) has gone away in C++.  Please try just switching to `std::string`.